lib/lightning/work_orders.ex

```diff
@@ -545,10 +547,7 @@ defmodule Lightning.WorkOrders do
         returning: true
       )
     end)
-    |> Multi.put(
-      :workflow_runs_count,
-      Enum.frequencies_by(runs, & &1.work_order.workflow_id)
-    )
+    |> workflow_runs_count(workorders)
```

Comment:
As much as possible we should make the `Multi` operation visible (the kind and name). 
For this reason would preserve `|> Multi.put(:workflow_runs_count,` and what about:
      Enum.frequencies_by(workorders, & &1.workflow_id)